test/unit/atomic_iteration_test.lua

```diff
@@ -134,39 +134,48 @@ end
 -- just check expirationd task continue work after conflicts
 function g.test_mvcc_vinyl_tx_conflict(cg)
     t.skip_if(cg.params.engine ~= 'vinyl', 'Unsupported engine')
-    -- TODO: Remove the line below when tarantool/expirationd#105 is resolved.
-    t.skip('Skipped until tarantool/expirationd#105 is resolved')
 
     for i = 1,10 do
         cg.space:insert({i, tostring(i), nil, nil, 0})
     end
 
     local updaters = {}
     for i = 1,10 do
-        local updater = fiber.create(function()
-            fiber.name(string.format("updater of %d", i), { truncate = true })
```

Comment:
Nit: The rule of thumb is always set a fiber name. This way it is easier to understand what is going wrong if something is going wrong.

Reply:
Ah, I see, it is set below. Usually we do it in a first line of a fiber function.

Reply:
Thank you! I'll try to remember.